setup.py

```diff
@@ -93,12 +109,10 @@ def run(self):
         "a P2P communications protocol",
         long_description=README,
         license='MIT',
-        # TODO: add author info
-        #author='',
-        #author_email='',
+        author='The Bitmessage Team',
+        author_email='surda@economicsofbitcoin.com',
```

Comment:
dev@bitmessage.org (new address, email hosting has been moved to protonmail a couple of weeks ago).